packages/utilities/psammead-styles/src/fonts.js

```diff
@@ -1,14 +1,188 @@
-// BBC News Article fonts
-const fontFamilyBase = ', Helvetica, Arial, sans-serif';
-
-export const FF_NEWS_SANS_LIT = `ReithSansNewsLight${fontFamilyBase}`;
-export const FF_NEWS_SANS_REG = `ReithSansNewsRegular${fontFamilyBase}`;
-export const FF_NEWS_SANS_REG_ITAL = `ReithSansNewsRegularItalic${fontFamilyBase}`;
-export const FF_NEWS_SANS_MDM = `ReithSansNewsMedium${fontFamilyBase}`;
-export const FF_NEWS_SANS_BLD = `ReithSansNewsBold${fontFamilyBase}`;
-
-export const FF_NEWS_SERIF_LIT = `ReithSerifNewsLight${fontFamilyBase}`;
-export const FF_NEWS_SERIF_REG = `ReithSerifNewsRegular${fontFamilyBase}`;
-export const FF_NEWS_SERIF_MDM = `ReithSerifNewsMedium${fontFamilyBase}`;
-export const FF_NEWS_SERIF_MDM_ITAL = `ReithSerifNewsMediumItalic${fontFamilyBase}`;
-export const FF_NEWS_SERIF_BLD = `ReithSerifNewsBold${fontFamilyBase}`;
+const baseFontUrl = 'https://gel.files.bbci.co.uk/r2.302/';
```

Comment:
Is this definitely a safe URL to use? I ask because we've done this sort of thing with IDT assets in the past, where it's tied to a version, but as and when new versions were released, old version assets would 'die'.

If it _is_ safe, can we link to the Confluence/JIRA/GitHub/GEL doc which says that?

Reply:
I think this is what #302 is about, so deemed it out of scope & just refactored this in such a way that they could changed easily. But, FWIW:
 
https://confluence.dev.bbc.co.uk/display/GELRD/BBC+Reith+font+hosting+on+the+gel-dev-account+AWS+account

This describes the reasoning behind the `gel.files.bbci.co.uk` link. However, I think the trade-offs described at the end of that sound fairly severe & I think #302's point on the caching is an important factor. If we're happy transitioning to `nav...`, I'd suggest you close #302 & add it as a criteria of the implementation ticket.

Reply:
Discussed in person, think we agree it belongs as a separate discussion.

Reply:
Sounds, iPlayer, CBBC, Sport & Homepage use fonts from `https://nav.files.bbci.co.uk`
Weather use fonts from `https://gel.files.bbci.co.uk/`